src/plugins/editor/components/editor.jsx

```diff
@@ -259,7 +259,7 @@ export default function makeEditor({ editorPluginsToRun }) {
       win.document.removeEventListener("click", this.onClick)
     }
 
-    componentWillReceiveProps(nextProps) {
+    UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
should not use React 16 method here.

Reply:
`componentWillReceiveProps` is trickier, as I think there might be underlying performance reasons for it (ace, large api definitions). I think it is safer for now to explicitly ignore this warning in eslint.

Reply:
Sure, indeed it does seem like this method is a bit more involved. I've fixed this now